applicationconfig/client/src/main/java/com/azure/applicationconfig/ConfigurationAsyncClientBuilder.java

```diff
@@ -26,7 +26,50 @@
 import java.util.Objects;
 
 /**
- * Provides configuration options for instances of {@link ConfigurationAsyncClient}.
+ * Builds instances of {@link ConfigurationAsyncClient} based on the configuration options stored in the builder when
+ * {@link ConfigurationAsyncClientBuilder#build()} is called.
+ *
+ * <p>
+ *     To build ConfigurationAsyncClients that can interact with Azure App Configuration the service endpoint and
+ *     authentication {@link HttpHeaders} are required. This information can be passed to the builder using two primary
+ *     means, first being a {@link ConfigurationClientCredentials} and second using a {@link HttpPipeline} and the
+ *     service endpoint.
+ * </p>
+ *
+ * <p>
+ *     The ConfigurationClientCredentials object contains the service endpoint and a method to retrieve the authentication
+ *     headers. {@link ConfigurationAsyncClientBuilder#credentials(ConfigurationClientCredentials)} sets the
+ *     {@code serviceEndpoint} member variable to {@link ConfigurationClientCredentials#baseUri()} and {@code credentials}
+ *     member variable to the passed ConfigurationClientCredentials. When building the ConfigurationAsyncClient the
+ *     builder will use default {@link HttpPipelinePolicy policies}, including the ConfigurationClientCredentials, to
+ *     construct a HttpPipeline that will be used by the client to interact with the service; a new pipeline will be
+ *     constructed every build.
+ *
+ * <pre>
+ * {@code ConfigurationClientCredentials credentials = new ConfigurationClientCredentials(connectionString);
+ *    ConfigurationAsyncClient.builder()
+ *        .credentials(credentials)
+ *        .build();}
+ * </pre>
+ * </p>
```

Comment:
This closing tag `</p>` is not necessary because the paragraph is followed by a `<pre>` and shows as an error when generating Javadocs.
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/p

Reply:
I would prefer to see the `</p>` placed before the opening `<pre>` code block. This should work as expected.

Reply:
Also, be sure to generate the javadoc on your machine to ensure everything is aligned appropriately. I typically don't put multi-line code starting on the same line as `{@code` - I normally start on a new line after that.